docs/source/about/metasyn_in_detail.rst

```diff
@@ -101,52 +102,58 @@ Model estimation starts with an appropriately pre-processed data frame. For ``me
      - -30
 
 
-For each data type supported by ``metasyn``, there is a set of candidate distributions that can be fitted to that data type (see Table below). To estimate the generative model of Equation, for each variable the software fits all compatible candidate distributions — by default with maximum likelihood estimation — and then selects the one with the lowest `AIC <https://springer.com/chapter/10.1007/978-1-4612-1694-0_15>`_.
+For each data type supported by ``metasyn``, there is a set of candidate distributions that can be fitted to that data
+type (see Table below). For each variable, the software fits all available distributions with the same variable type.
+From all those fits, the distribution with the lowest `AIC <https://springer.com/chapter/10.1007/978-1-4612-1694-0_15>`_ is chosen.
 
 .. list-table::
    :header-rows: 1
 
    * - Variable type
      - Data type
-     - example
-     - candidate distributions
+     - Example values
+     - Example distribution
    * - continuous
      - float
      - 1.0, 2.1, ...
-     - UniformDistribution, NormalDistribution, ...
+     - UniformDistribution
    * - discrete
      - int
      - 1, 2, ...
      - DiscreteUniformDistribution
    * - categorical
      - pl.Categorical
-     - gender, country
+     - Male, Female, Male, Female
```

Comment:
```suggestion
     - Male, Female, Non-binary, Female
```

Tiny inclusivity change

Reply:
Thanks! I have changed it to Yes/No/Maybe. That should suffice as well I think.